docs/source/certificates/certificates.md

```diff
@@ -0,0 +1,173 @@
+|----------------|-----|---------------------------|-----------|---|
+| Ordering service organization channel admin certificate<br><br>Type=admin| The admin certificate used to administer the ordering service and submit or approve channel updates. The associated admin identity is enrolled with the organization CA before the node and organization MSP are created.| Depends on implementation&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br><br><code>ordererca/ <br />└── ordereradmin <br />&nbsp;&nbsp;└── msp <br />&nbsp;&nbsp; &nbsp;&nbsp;├── admincerts <br />&nbsp;&nbsp; &nbsp;&nbsp;│    &nbsp;&nbsp;└── cert.pem <br />&nbsp;&nbsp; &nbsp;&nbsp;├── cacerts <br />&nbsp;&nbsp; &nbsp;&nbsp;│    &nbsp;&nbsp;└── localhost-7053.pem <br />&nbsp;&nbsp; &nbsp;&nbsp;├── keystore <br />&nbsp;&nbsp; &nbsp;&nbsp;│    &nbsp;&nbsp;└── key.pem <br />&nbsp;&nbsp; &nbsp;&nbsp;├── signcerts <br />&nbsp;&nbsp; &nbsp;&nbsp;│    &nbsp;&nbsp;└── cert.pem <br />&nbsp;&nbsp; &nbsp;&nbsp;└── user <br /></code>| Transactions can continue to process  successfully.<br><br>Cannot modify channels from a client application or manage the orderer from the console.| https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-org0-s-admin|
+| Peer service Organization channel admin certificate<br><br>Type=admin| The admin certificate used to administer the peer and install smart contracts on the peer. Can be configured to manage the  application channel that the peer belongs to. The associated admin user is registered with the organization CA before the peer and organization MSP are created.| Depends on implementation<br><br><code>org1ca/ <br />└── org1admin <br />&nbsp;&nbsp;└── msp <br />&nbsp;&nbsp; &nbsp;&nbsp;├── admincerts <br />&nbsp;&nbsp; &nbsp;&nbsp;│   &nbsp;&nbsp;└── cert.pem <br />&nbsp;&nbsp; &nbsp;&nbsp;├── cacerts <br />&nbsp;&nbsp; &nbsp;&nbsp;│   &nbsp;&nbsp;└── localhost-7053.pem <br />&nbsp;&nbsp; &nbsp;&nbsp;├── keystore <br />&nbsp;&nbsp; &nbsp;&nbsp;│   &nbsp;&nbsp;└── key.pem <br />&nbsp;&nbsp; &nbsp;&nbsp;├── signcerts <br />&nbsp;&nbsp; &nbsp;&nbsp;│   &nbsp;&nbsp;└── cert.pem <br />&nbsp;&nbsp; &nbsp;&nbsp;└── user <br /> </code>| Transactions can continue to process successfully. <br> <br>Cannot install new smart contracts from a client application or manage the peer from the console.| https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-org1-s-admin |
+
+### Client Certificates
+
+Client certificates are created by applications, using the Fabric SDKs or command line CA utilities. These certificates expire after one year if using the Hyperledger Fabric CA default settings.   Certificates must be re-enrolled before expiration, or the client application will not be able to connect to the Fabric nodes.
```

Comment:
It would be good to also say who signs the certificates, since in order to have a valid one, both the client _and_ the CA have to cooperate.

Reply:
Good point, yes, let's be more specific. Also because there's yet another type of client certificate, namely, the one used by whoever runs the client SDK (in case there's no gateway).

Reply:
Will need to be reviewed again. ... There is no generation statement or location statement for the 3 client cert types.